admin/src/components/AbuseReportsPopUp/ModelFooter.js

```diff
@@ -0,0 +1,31 @@
+import { ModalFooter as StrapiModalFooter, colors } from 'strapi-helper-plugin';
```

Comment:
Some small issues:
1. This filename has a typo
2. In chrome devtools console there are some errors/warnings, like missing key or required property has undefined value